### PR TITLE
add debian dependencies

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Add python deps to debian client instructions.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.1
- [ ] Manager-4.0

# Links

- Fixes https://github.com/SUSE/spacewalk/issues/14331
- Bug https://bugzilla.suse.com/show_bug.cgi?id=1183649

### CODE REVIEW

File: modules/client-configuration/pages/clients-debian.adoc [hunk 1]
```diff
@@ -26,15 +26,15 @@ Bootstrapping can be used with {debian} clients for performing initial state run
 
 Some preparation is required before you can register {debian} clients to the {productname} Server:
 
-* If you are using Debian{nbsp}9, install the ``apt-transport-https`` package on the client before you attempt to register.
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-debian.adoc:29]: One package here...

File: modules/client-configuration/pages/clients-debian.adoc [hunk 2]
```diff
@@ -26,15 +26,15 @@ Bootstrapping can be used with {debian} clients for performing initial state run
...
+* If you are using Debian{nbsp}9, install the required packages on the client before you attempt to register.
   On the client, at the command prompt, as root, run:
 +
 ----
-apt install apt-transport-https
+apt install apt-transport-https python-apt python3-apt
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-debian.adoc:33]: ... and three here.

I'd keep "install the required packages" at line 29, and just the three packages at line 33, instead of duplicating the list.

**Review by juliogonzalez — CHANGES_REQUESTED** [review]

File: modules/client-configuration/pages/clients-debian.adoc [hunk 3]
```diff
@@ -26,15 +26,15 @@ Bootstrapping can be used with {debian} clients for performing initial state run
...
 ----
 
 * Ensure DNS is correctly configured and provides an entry for the client.
-    Alternatively, you can configure the ``/etc/hosts`` files on both the {productname} Server and the client with the appropriate entries.
+  Alternatively, you can configure the ``/etc/hosts`` files on both the {productname} Server and the client with the appropriate entries.
 * The client must have the date and time synchronized correctly with the {productname} Server before registration.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-debian.adoc:38]:
```suggestion
* The client must have the date and time synchronized with the {productname} Server before registration.
```
drive-by edit.

**Review by keichwa — APPROVED** [review]

File: modules/client-configuration/pages/clients-debian.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/client-configuration/pages/clients-debian.adoc:33]: I ... did?

File: modules/client-configuration/pages/clients-debian.adoc (hunk 3 above)
Comment (Loquacity) [reply modules/client-configuration/pages/clients-debian.adoc:38]: I think that exists in all the client files ...

File: modules/client-configuration/pages/clients-debian.adoc (hunk 2 above)
Comment (juliogonzalez) [reply modules/client-configuration/pages/clients-debian.adoc:33]: Yes, you did. I think I am becoming daltonic or something. Either that or I need new glasses.

PR is fine, sorry about the noise (for some reason I thought you were adding `apt-transport-https` to line 29, when in fact you we are removing it.

**Review by juliogonzalez — APPROVED** [review]

File: modules/client-configuration/pages/clients-debian.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/client-configuration/pages/clients-debian.adoc:33]: OK, I thought I was going mad for a bit there! Maybe go see an optometrist? :wink:

**Comment by Loquacity** [comment]
:heavy_check_mark: Backport 4.1: https://github.com/uyuni-project/uyuni-docs/pull/891
